### PR TITLE
Mark warning messages with W instead of *

### PR DESCRIPTION
**Pull request by mastizada**
It was discussed in PR #1432

### CODE REVIEW

**Comment by mastizada** [comment]
Just saw issue #1434, I think it can be better to leave that for new contributor.

**Comment by pocmo** [comment]
@mastizada Thanks! 👍 Let's wait some days and if no other contributor picks it up then I'll merge your patch. :)

File: tools/l10n/check_translations.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def missing_source_exception(lang, code):
 
 def count_mismatch_warning(lang, code):
     # number of same placeholders used in source and target do not match
-    print(" * [{lang}/strings.xml] number of placeholders not matching, key: {code}".format(
+    print(" W [{lang}/strings.xml] number of placeholders not matching, key: {code}".format(
```
Comment (pocmo) [inline tools/l10n/check_translations.py]: I think we can be more verbose and actually scream "WARNING: " or "ERROR: " into the face of the script user :)

**Comment by mastizada** [comment]
Example output:
```
Checking for placeholders in translation files...
 * ERROR: [values-az/strings.xml] ...
 WARNING: [values-eo/strings.xml] ...
 WARNING: [values-pl/strings.xml] ...
```

**Comment by crud3** [comment]
@pocmo  I'd like to pick it up!

I'm not sure about how useful it is though, as someone already commented on #1434 and this PR here already exists with a correct solution and everything :)
 But if you think it's okay I'll just do it.

**Comment by pocmo** [comment]
@crud3 Do you want to write Python code or are you interested in contributing in general (Android/Java)? We could try to find an other issue for you!

**Comment by crud3** [comment]
@pocmo  If you have some tasks to delegate, I would be glad to help :) 
I'm interested in both. Right now I'm have a lot more experience writing Java code, but I'm trying to learn Python anyways and have written some more or less small scripts in the past

**Pull request closed without merging**